packages/wdio-reporter/src/index.js

```diff
@@ -155,6 +155,7 @@ export default class WDIOReporter extends EventEmitter {
         this.on('runner:end',  /* istanbul ignore next */ (runner) => {
             rootSuite.complete()
             this.runnerStat.failures = runner.failures
+            this.runnerStat.retries = runner.retries
```

Comment:
1) Add this into doc https://github.com/webdriverio/webdriverio/tree/master/packages/wdio-reporter#runner-events
2) Is it possible to add info about retries to `runner:start` event? In fact I want to have info - is current runner doing spec retry or not. here this event emited
`https://github.com/webdriverio/webdriverio/blob/6c55c8afe2be480c18784ba42189ebc31d0585eb/packages/wdio-runner/src/index.js#L94`. 
It is usefull for some powerfull reporters like allure\reportportal.

Reply:
1. Will do.
2. Have a look at https://github.com/webdriverio/webdriverio/pull/3381#issuecomment-465943670 and the roughly 6 comments preceding it. When a new runner starts, if its runner id matches that of an earlier runner, you know it's a retry.

Reply:
1. Thank you!
2. It is nice that `cid` is the same. But we can't save info which cid's already executed(and processed by reporter) because on every `runner.run()` we create new instance of reporter see
https://github.com/webdriverio/webdriverio/blob/91b7863149d925dfea2fa9fb3d825e1e90743a3d/packages/wdio-runner/src/index.js#L61
So it will be nice to have such info from runner start event.

Mayby I miss something in runner\report relationships @christian-bromann ?

Reply:
Summon @christian-bromann again

Reply:
@BorisOsipov is right, `runner.run()` creates a new worker instance and therefor creates a new process where the reporter is fresh initialized with no info about any previous rerun. Let's add the retry number to the `runner:start`